cmd/service-controller/policy_controller.go

```diff
@@ -347,15 +347,19 @@ func (c *PolicyController) validateExposeStateChanged() {
 		for _, target := range service.Targets {
 			svcNamespace := utils.GetOrDefault(target.Namespace, c.cli.GetNamespace())
 			targetType := c.inferTargetType(target, svcNamespace)
-			res := c.validator.ValidateExpose(targetType, target.Name)
+			targetName := target.Name
+			if targetType == "service" && svcNamespace != c.cli.GetNamespace() {
```

Comment:
If we are preventing the target namespace being specified when target tpe is 'service', do we need this here?

Reply:
We are preventing the `--target-namespace` flag from being provided through the CLI when the target type is
service, but allowing users to bind a service from another namespace by qualifying the service.

With latest version of Skupper, the namespace field of a target in skupper-services is never empty.

So if we populate the "service" field as a qualified value, the namespace field would be populated incorrectly.

My inclination is to keep the namespace field being populated, as is, and when a service is qualified, we
split it and store the qualified namespace at the namespace field and leave the service field unqualified there.

Do you think it is better to change this current behavior and leave the "namespace" field empty in the targets
when target type is service?

Reply:
I will make this change then you ping you again for another round of review.
And I believe your other questions might be answered once I have it done.

Reply:
Done.